example/src/CSE.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Button, SafeAreaView } from 'react-native';
+import { NativeModules } from 'react-native';
+import { ENVIRONMENT } from './Configuration';
+import Styles from './Styles';
+
+const { AdyenCSE } = NativeModules;
+
+const CseView = () => {
+  const tryEncryptCard = async () => {
+    let unencryptedCard = {
+      number: '5454 5454 5454 5454',
+      expiryMonth: '03',
+      expiryYear: '2030',
+      cvv: '737',
+    };
+    let publicKey = ENVIRONMENT.publicKey;
+    try {
+      const encryptedCard = await AdyenCSE.encryptCard(
+        unencryptedCard,
+        publicKey
+      );
+      console.log(JSON.stringify(encryptedCard));
+    } catch (e) {
+      console.error(e);
+    }
+  };
```

Comment:
You can wrap the declared functions in `useCallback` hook - this way, they aren't recreated in memory if the Component re-renders.

```jsx
const tryEncryptCard = useCallback(async () => {
...
}. []);
```